package.json

```diff
@@ -54,10 +54,14 @@
     "./writer": {
       "browser": "./lib/writer-browser.js",
       "import": "./lib/writer.js"
+    },
+    "./buffer-writer": {
```

Comment:
you're going to need to add an equivalent block in `typesVersions` so consumers can find the types for it (i.e. avoiding the `lib` indirection)